config/routes.rb

```diff
@@ -2,6 +2,14 @@
 
 Rails.application.routes.draw do
   devise_for :user, controllers: { registrations: "users/registrations" }
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
   root to: "root#index"
+
+  get "/articles/new", to: "article#new", as: "new_article"
+  post "/articles/new", to: "article#new", as: "articles"
```

Comment:
Usually it is just /articles
```suggestion
  post "/articles", to: "article#create", as: "articles"
```